src/backend/marsha/core/serializers.py

```diff
@@ -299,6 +302,31 @@ class Meta:  # noqa
     thumbnail = ThumbnailSerializer(read_only=True, allow_null=True)
     urls = serializers.SerializerMethodField()
     is_ready_to_show = serializers.BooleanField(read_only=True)
+    disabled_use_subtitle_as_transcript = serializers.SerializerMethodField()
+
+    def get_disabled_use_subtitle_as_transcript(self, obj):
+        """Compute if use_subtitle_as_transcript behavior is disabled.
+
+        Parameters
+        ----------
+        obj : Type[models.Video]
+            The video that we want to serialize
+
+        Returns
+        -------
+        Boolean
+            If there is at least one transcript ready to be shown the method will return False.
+            Returns True otherwise.
+        """
+        timed_text_tracks = obj.timedtexttracks.all()
+
+        transcripts = [
+            timed_text_track
+            for timed_text_track in timed_text_tracks
+            if timed_text_track.mode == "ts"
+        ]
+
+        return len(transcripts) > 0
```

Comment:
It should be doable in 1 query. Something like:
```suggestion
        return obj.timedtexttracks.filter(mode="ts").exists()
```

You should also add a test with a video that has several timed text tracks and make sure that only 1 query is issued (using assertNumQueries).